assets/color.css

```diff
@@ -0,0 +1,6 @@
+h2, h3, h4, h5, h6 {
+    color: #8D9D92 !important;
```

Comment:
the '!important' is to override another CSS styling choice right?

Reply:
yes